roxctl/deployment/check/check.go

```diff
@@ -162,8 +165,16 @@ func (d *deploymentCheckCommand) Construct(_ []string, cmd *cobra.Command, f *pr
 }
 
 func (d *deploymentCheckCommand) Validate() error {
-	if _, err := os.Open(d.file); err != nil {
-		return common.ErrInvalidCommandOption.CausedBy(err)
+	if d.file != "" {
```

Comment:
Another reason to remove `--file` in favor of `--files` only is to avoid this odd behavior if both are passed but only one considered.

Reply:
See comment above, I think there is still a reason to merge this. Otherwise, we should at least show a warning if both parameters are used together.

Reply:
I ended up merging the two of them into `--file`, preserving old behavior.